test/coreneuron/test_direct.py

```diff
@@ -40,6 +40,7 @@ def test_direct_memory_transfer():
     pc.psolve(h.tstop)
     tran = [h.t, h.soma(.5).v, h.soma(.5).hh.m]
 
+    assert(v.eq(vstd))
```

Comment:
The original test did not require equality but only within tolerance in line 45.  I guess the equality in line 43 in terms of hoc_epsilon = 1e-11 was too tight.

Reply:
This is a leftover artifact from my trajectory gdb debugging. Should I revert it?

Reply:
Yes. On my machine that failed. I should check again but it would seem that either line 45 or line 43 should go.

Reply:
Darn. I must be doing something wrong and now I have to figure out what it is.  ```make test``` gives:
```
The following tests FAILED:
	  4 - basic_tests (Failed)
	  5 - rxd_tests (Failed)
	  6 - rxd_mpi_tests (Failed)
	  8 - coreneuron_direct_py (Failed)
	 10 - coreneuron_spikes_py (Failed)
	 12 - coreneuron_fornetcon_py (Failed)
	 13 - coreneuron_datareturn_py (Failed)
	 14 - coreneuron_test_units_py (Failed)
	 15 - coreneuron_test_natrans_py (Failed)
```

Reply:
Looks like this is problematic: https://github.com/neuronsimulator/nrn/issues/1071 
We should revert.